### PR TITLE
Feature/zoom bar/visual

### PR DESCRIPTION
**Pull request by Delfil**
Scroll bar with interest display.

### CODE REVIEW

**Comment by Delfil** [comment]
Edge thickness does not longer work?

File: src/main/java/tudelft/ti2806/pl3/visualization/GraphView.java [hunk 1]
```diff
@@ -309,4 +302,68 @@ public void centerOnNode(DataNode node) throws NodeNotFoundException {
...
+		if (zoomCenterSet == false) {
+			setZoomCenter(0);
+			setOffsetToCenter();
+			setZoomCenter(offsetToCenter);
+			zoomCenterSet = true;
+		}
+	}
+
+	@Override
+	public void componentMoved(ComponentEvent e) {
```
Comment (TomBrouws) [inline src/main/java/tudelft/ti2806/pl3/visualization/GraphView.java]: What's the use of all these empty overrides? Should they not at least have a call to super?

File: src/main/java/tudelft/ti2806/pl3/zoomBar/ZoomBarController.java [hunk 2]
```diff
@@ -33,4 +39,37 @@ public GraphController getGraphController() {
...
+		double size = graphController.getGraphView().getGraphDimension();
+		float factor = (float) e.getX()
+				/ (float) ScreenSize.getInstance().getWidth();
+		float newPos = (float) (factor * size);
+		graphController.moveView(newPos);
+	}
+
+	@Override
+	public void mousePressed(MouseEvent e) {
+
```
Comment (TomBrouws) [inline src/main/java/tudelft/ti2806/pl3/zoomBar/ZoomBarController.java]: Same comment as before on empty overrides

File: src/main/java/tudelft/ti2806/pl3/zoomBar/ZoomBarView.java [hunk 3]
```diff
@@ -37,4 +115,29 @@ public Component getPanel() {
...
+		repaint();
+	}
+
+	@Override
+	public void componentResized(ComponentEvent e) {
+		moved();
+	}
+
+	@Override
+	public void componentMoved(ComponentEvent e) {
```
Comment (TomBrouws) [inline src/main/java/tudelft/ti2806/pl3/zoomBar/ZoomBarView.java]: Same comment as before on empty overrides

**Comment by TomBrouws** [comment]
Looks good except for my comments! For some reason clicking to navigate in the zoombar works on this branch

File: src/main/java/tudelft/ti2806/pl3/visualization/GraphView.java (hunk 1 above)
Comment (borismattijssen) [inline src/main/java/tudelft/ti2806/pl3/visualization/GraphView.java]: It implements an interface which requires me tho implement these methods.

**Pull request merged**